<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestpageComputer</class>
 <widget class="QMainWindow" name="TestpageComputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>599</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="0">
     <widget class="QLabel" name="yes">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>458</width>
         <height>533</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="10" column="0" colspan="3">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>15</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QRadioButton" name="radioButton_5">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>A</string>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="A_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>ENIAC</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radioButton_6">
              <property name="text">
               <string>B</string>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="B_2">
              <property name="text">
               <string>EDVAC </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radioButton_7">
              <property name="text">
               <string>C</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="C_2">
              <property name="text">
               <string>EDSAC </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="topMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radioButton_8">
              <property name="text">
               <string>D</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="D_2">
              <property name="text">
               <string>UNIVAC</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="11" column="0" colspan="3">
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>Next - अगला </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="9" colspan="3">
         <widget class="QTextBrowser" name="sandeep_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;पहला सक्रियात्मक इलेक्ट्रॉनिक अंकीय कम्प्यूटर है-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="no">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
